.travis.yml

```diff
@@ -11,11 +11,11 @@ env:
 
 # installing dependencies
 before_install:
+  - pip install --upgrade pytest
```

Comment:
Rather than upgrading pytest here, add `--upgrade` to the dev-requirements below.

I also see that dev-requirements is added twice. It doesn't need to be in the `install` step as well as the `before_install` step